package.json

```diff
@@ -60,7 +64,8 @@
     "storybook:build": "lerna --scope @carto/react-ui exec -- npm run storybook:build",
     "storybook:publish": "lerna --scope @carto/react-ui exec -- npm run storybook:publish",
     "publish:prerelease": "yarn run build:clean && lerna publish --dist-tag prerelease --force-publish --preid rc",
-    "publish:release": "yarn run build:clean && lerna publish --force-publish"
+    "publish:release": "yarn run build:clean && lerna publish --force-publish",
+    "postinstall": "patch-package"
```

Comment:
This comment https://github.com/CartoDB/carto-react/pull/417/files#diff-337ead04a20e03f578a3166d4928bcf79a396bfd9df9e750662aeb689a5d5aceR2